test/decorators/provider_decorator_test.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class ProviderDecoratorTest < Draper::TestCase
+
+  def setup
+    provider = FactoryBot.create(:provider_account)
+    @decorator = ProviderDecorator.new provider
+    @decorator.stubs(:service_plans_management_visible?).returns(true)
+  end
+
+  test 'new_application_form_data in Dashboard context' do
+    data = {}
+    @decorator.stubs(:audience_context_new_application_form_data).returns(data).once
```

Comment:
Instead of `stubs ... once` maybe `expects ... once`
Stubbing ≠ mocking

Reply:
> Also assignment of `data` is not needed here
It is in order to avoid repetition

Reply:
Yes well you repeat it in other tests always :) but here is not necessary, it is used only once. See below tests they are the same

Reply:
Ah, you're right. There used to be repetition here but I ended up simplifying it.